src/Lecture2.hs

```diff
@@ -58,7 +62,9 @@ lazyProduct = error "TODO"
 "ccaabb"
 -}
 duplicate :: [a] -> [a]
-duplicate = error "TODO"
+duplicate []     = []
+duplicate [x]    = [x,x]
```

Comment:
You can remove this case as it's already covered by the other too

Reply:
did it. thank you